inventory_email_support/fixtures/email_templates.json

```diff
@@ -1,25 +1 @@
% endfor %}\r\n</ul>\r\n<br />\r\n</p>\r\n<p> Please contact a staff member if you have any questions. </p> <p> Best Regards, <br /> ECE Inventory System Staff </p>", "language": "", "default_template": null}}, {"model": "post_office.emailtemplate", "pk": 5, "fields": {"name": "convert_loan_to_disbursement", "description": "", "created": "2017-03-23T19:42:53.544Z", "last_updated": "2017-03-24T03:33:22.428Z", "subject": "{{ subject }}", "content": "", "html_content": "<p> Hi {{ name }}, </p> <p> The following item that was loaned to you has now been converted to disbursement. The item name and quantity converted are shown below.</p>\r\n<p>\r\n{{ item_name }} ({{ quantity }})\r\n</p>\r\n<p> Please contact a staff member if you have any questions. </p> <p> Best Regards, <br /> ECE Inventory System Staff </p>", "language": "", "default_template": null}}, {"model": "post_office.emailtemplate", "pk": 6, "fields": {"name": "loan_reminder", "description": "", "created": "2017-03-24T03:35:56.371Z", "last_updated": "2017-03-24T05:08:10.821Z", "subject": "{{ subject }}", "content": "", "html_content": "<p> Hi {{ name }}, </p> {% if prepended_body %} <p> {{ prepended_body }} </p> {% endif %}\r\n<p> Items Loaned to You: <br />\r\n<ul> {% for loan in  loaned_items %}\r\n<li> {{ loan.item.name }} ({{ loan.quantity }})</li>\r\n{% endfor %}\r\n</ul>\r\n<br />\r\n</p>\r\n<p> Please contact a staff member if you have any questions. </p> <p> Best Regards, <br /> ECE Inventory System Staff </p>", "language": "", "default_template": null}}, {"model": "post_office.emailtemplate", "pk": 7, "fields": {"name": "minimum_stock_reached", "description": "", "created": "2017-04-01T06:23:36.446Z", "last_updated": "2017-04-01T06:35:33.263Z", "subject": "{{ subject }}", "content": "", "html_content": "<p> Hi, </p> <p> The following Item has reached it's minimum quantity </p>\r\n<style type=\"text/css\">\r\n.tg  {border-collapse:collapse;border-spacing:0;}\r\n.tg td{font-family:Arial, sans-serif;font-size:14px;padding:10px 5px;border-style:solid;border-width:1px;overflow:hidden;word-break:normal;}\r\n.tg th{font-family:Arial, sans-serif;font-size:14px;font-weight:normal;padding:10px 5px;border-style:solid;border-width:1px;overflow:hidden;word-break:normal;}\r\n.tg .tg-y7dx{background-color:#fe0000;color:#333333;vertical-align:top}\r\n.tg .tg-yw4l{vertical-align:top}\r\n.tg .tg-y0xi{background-color:#32cb00;vertical-align:top}\r\n</style>\r\n<table class=\"tg\" border=\"2\">\r\n  <tr>\r\n    <th class=\"tg-yw4l\">Name</th>\r\n    <th class=\"tg-yw4l\">Current Quantity</th>\r\n    <th class=\"tg-yw4l\">Threshold Quantity</th>\r\n  </tr>\r\n  <tr>\r\n    <td class=\"tg-yw4l\">{{ name }}</td>\r\n    <td class=\"tg-y0xi\">{{ quantity }}</td>\r\n    <td class=\"tg-y7dx\">{{ minimum_stock }}</td>\r\n  </tr>\r\n</table>\r\n<p> Please contact a staff member if you have any questions. </p> <p> Best Regards, <br /> ECE Inventory System Staff </p>", "language": "", "default_template": null}}]
```

Comment:
Can the message say something more like this?

The quantity of the following item has gone below its configured threshold.